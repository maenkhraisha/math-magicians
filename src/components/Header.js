import React from 'react';
import { Link } from 'react-router-dom';

function Header() {
  return (
    <div className="header">
      <h1 className="header-title">Math Magicians</h1>
      <ul className="header-nav">
        <li><Link className="header-nav-item" to="/">Home</Link></li>
        <span> | </span>
        <li><Link className="header-nav-item" to="/Quote">Quote</Link></li>
        <span> | </span>
        <li><Link className="header-nav-item" to="/Calculator">Calculator</Link></li>
      </ul>
    </div>
  );
}

export default Header;
