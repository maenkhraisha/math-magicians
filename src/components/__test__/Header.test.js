import React from "react";
import renderer from "react-test-renderer";

import Header from "../Header";

it("Render correctly", () => {
    const tree = renderer.create(<header />).toJSON();
    expect(tree).toMatchSnapshot();
});
