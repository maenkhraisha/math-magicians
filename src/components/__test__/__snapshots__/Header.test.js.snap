// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render correctly 1`] = `<header />`;
